src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,43 @@
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
 def merge( arrA, arrB ):
+    
+    if len(arrA) == 0:
+        return arrB
+    if len(arrB) == 0:
+        return arrA
+    
     elements = len( arrA ) + len( arrB )
     merged_arr = [0] * elements
-    # TO-DO
+
+    x, y = 0, 0
+    for i in range(elements):
+        if len(arrA) == x:
+            for j in range(y, len(arrB)):
+                merged_arr[i + j - y] = arrB[j]
```

Comment:
its was a little hard to wrap my head around all the indices here at first glance.  perhaps there are some ways to make this more readable with how we define the range, the name of the indices, or add short comments.

In the end this is a great way to copy over the remaining elements and return the result.  Good job.